ui/DesignSystem/Component/Badge.svelte

```diff
@@ -0,0 +1,17 @@
+<script>
+  export let variant = null;
+  export let style = null;
+
+  const caption = String(variant);
+</script>
+
+<style>
+  .badge {
+    color: var(--color-primary);
+    background: var(--color-primary-level-1);
+    border-radius: 4px;
+    padding: 0.125rem 0.25rem;
```

Comment:
The height of the badge should be 24px. It's currently 21.
![image](https://user-images.githubusercontent.com/4406983/91294491-e5d25200-e799-11ea-83de-f960fce1acec.png)